Sources/Nimble/Matchers/BeIdenticalTo.swift

```diff
@@ -19,6 +19,11 @@ public func !==<T: AnyObject>(lhs: Expectation<T>, rhs: T?) {
     lhs.toNot(beIdenticalTo(rhs))
 }
 
+/// Alias for "beIdenticalTo"
```

Comment:
While it's nice to note that it's an alias to `beIdenticalTo`, it's still useful to provide a one-liner description on what it actually do.